### PR TITLE
Update first chapter PT translation.

### PR DESCRIPTION
**Pull request by bbgx**
Updated text from the first chapter in Portuguese.

### CODE REVIEW

File: _chapters/pt/who-is-this-guide-for.md [hunk 1]
```diff
@@ -12,6 +12,26 @@ Esse guia foi feito para desenvolvedores full-stack ou desenvolvedores que desej
...
+Pessoalmente, a ideia do Serverless foi uma enorme revelação para nós e isso nos fez criar um guia com qual poderíamos compartilhar o que aprendemos. Você pode saber mais sobre nós [**aqui**]({% link about/index.html %}). E [veja alguns exemplos de pessoas que construíram aplicações com SST clicando aqui]({% link showcase.md %})
 
-Pessoalmente, a ideia do Serverless foi uma enorme revelação para nós e isso nos fez criar um guia com qual poderíamos compartilhar o que aprendemos. Você pode saber mais sobre nós [**aqui**]({% link about/index.html %}). E [veja alguns exemplos de pessoas que construiram aplicações com a stack Serverless]({% link showcase.md %})
+Por hora, apenas vamos abordar o desenvolvimento com JavaScript/TypeScript. Futuramente talvez abordemos outras linguagens e ambientes. Porém, para começar, nós achamos muito benéfico para um desenvolvedor full-stack utilizar apenas uma linguagem (JavaScript) e ambiente (NodeJs) para a construção de uma aplicação completa.
```
Comment (ViniciusAndr) [inline _chapters/pt/who-is-this-guide-for.md:17]: (JavaScript) at the end of the line was changed to (TypeScript) in the last change in the EN chapter and I think that (NodeJs) can be changed to (Node.js) to be the same as the EN version.

File: _chapters/pt/who-is-this-guide-for.md [hunk 2]
```diff
@@ -12,6 +12,26 @@ Esse guia foi feito para desenvolvedores full-stack ou desenvolvedores que desej
...
+
+### Por que Typescript
```
Comment (ViniciusAndr) [inline _chapters/pt/who-is-this-guide-for.md:19]: Typescript can be changed to TypeScript to keep the default.

File: _chapters/pt/who-is-this-guide-for.md [hunk 3]
```diff
@@ -12,6 +12,26 @@ Esse guia foi feito para desenvolvedores full-stack ou desenvolvedores que desej
...
+
+Nós usamos TypeScript desde o frontend, backend e até a criação da nossa infraestrutura. Se você não estiver familiar com TypeScript talvez você esteja pensando por que tipagem estática importa.
+
+Uma grande vantagem em usar tipagem estática em todo o código é que seu editor de código consegue autocompletar e mostrar opções inválidas no seu código. Isso é muito útil quando você esta começando. Porém, isso também pode ser útil quando você está configurando sua infraestrutura através de código.
+
+Deixando toda essa benevolência do autocomplete de lado, tipagem estática acaba sendo um ponto critico para ajudar na manutenibilidade de um projeto. Isso importa muito se você pretende trabalhar no mesmo projeto, com o mesmo código, por anos.
```
Comment (ViniciusAndr) [inline _chapters/pt/who-is-this-guide-for.md:25]: Critico is written Crítico when talking about a situation that need attention or an important topic.

**Comment by jayair** [comment]
Thank you!

**Pull request merged**